pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -16,6 +16,8 @@ social:
   @header["social"]["og:description"] = hoc_s(:social_hoc2018_global_movement)
   @header["social"]["og:image"] = 'https://' + request.host + '/images/social-media/code-2018-creativity.png'
 
+  launch_ai = DCDO.get('launch_ai', nil)
```

Comment:
this needs a `-` for HAML to read this as a variable assignment, right?

```suggestion
  - launch_ai = DCDO.get('launch_ai', nil)
```

Reply:
It's still in the `:ruby` block from line 10 (which you can't see in this diff 🙂)

Reply:
woops, you're right!